### PR TITLE
DS 1 to 24

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Corrections in "Palaeography" and occasional other minor corrections.

### CODE REVIEW

File: DayrSuryan/DSEthiop13.xml [hunk 1]
```diff
@@ -182,7 +182,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
-                    <seg type="script">Second half of the nineteenth-/early twentieth-century script of chant manuscripts.</seg>
+                    <seg type="script">Second half of the nineteenth-/early twentieth-century small script of
+                      manuscripts.</seg>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop13.xml:186]: no "chant" anymore? To me, it was clearer with it.

File: DayrSuryan/DSEthiop8.xml [hunk 2]
```diff
@@ -715,7 +715,7 @@ The Coptic patriarch <persName ref="PRS3258CyrilIV">Cyril IV (in tenure <date>18
                           <roleName type="title"><hi rendition="simple:italic">ʾAbbā</hi></roleName> Salāmā (in tenure <date>1841–1867</date>)</persName> are mentioned in a supplication
                        on <locus target="#6r"/>. The name of
                        <persName ref="PRS11944GabraM">Gabra Māryām</persName>, probably the commissioner, is mentioned
-                       in the supplications (eg., <locus target="#2r"/>, <locus target="#6v"/>). The manuscript
+                       in the supplications on <locus target="#2r"/> and <locus target="#6v"/>. The manuscript
```
Comment (thea-m) [inline DayrSuryan/DSEthiop8.xml:718]: I feel that this modification changes the sense. The first version implies that there are more than two supplications, the modification that these are the only ones or the only ones in which he is mentioned. Is this the case?

**Review by thea-m — CHANGES_REQUESTED** [review]
some questions of understanding

File: DayrSuryan/DSEthiop13.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop13.xml:186]: ok, restituted.

File: DayrSuryan/DSEthiop8.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop8.xml:718]: modified

**Review by thea-m — APPROVED** [review]

**Pull request merged**